data-esintl.js

```diff
@@ -22,7 +22,7 @@ exports.tests = [
         safari10: true,
         opera10_50: false,
         node0_12: true,
-        duktape2_0: false,
+        duktape1_0: false,
```

Comment:
I did some "result optimization" myself on the beginning, but it is not the way it is supposed to work.  You should keep here the results for the versions that were actually tested, and let the compiler guess the others.  So you are not supposed to remove correct results, only add new ones.

Reply:
I've received a little contradicting feedback on this: previously when I added results for 2.1 that were the same as for 2.0 -- both tested explicitly -- I got feedback that the 2.1 results shouldn't be added if they are not different from 2.0.

Reply:
Interesting.  Maybe it's a major/minor thing then?  Let's hear from the committers then.

Reply:
Also the results for at least Duktape 2.2 (and possibly Duktape 1.3 and 1.4) were added (by someone else) without actually running tests. So some of the 2.2 results were incorrect before the pull. I'm not sure if the 1.x data is correct either (i.e. whether it has actually been tested or is just based on release notes or something like that).

Reply:
@afmenez It seems that the previous pull with 2.2 results was actually from you - can you confirm whether the 1.x results are based on actual test runs or release notes only? If the latter, I could try to run the tests with 1.x to make the results exact?

Reply:
Yes, all the data I added was based on the release notes.  You can sure run the tests if you can and correct everything that is wrong!

Reply:
Ok, I'll do that in a separate pull later on if these results are interesting for old versions :)

Reply:
Actually it would be great if you could add the actual results for 1.x as you did before.  There's a separate PR #1131 that changes the default from "false" to "unknown", and it would affect most of the results for 1.x if they are not explicitly set.

Reply:
Hmm, so the most important thing would be to get 1.0 results for the full test set, because those are inherited forwards automatically.

Reply:
Exactly.